src/tests/resize.test.tsx

```diff
@@ -89,10 +90,9 @@ describe("resize element with aspect ratio when SHIFT is clicked", () => {
     fireEvent.pointerUp(canvas);
 
     expect(renderScene).toHaveBeenCalledTimes(5);
-    const elements = renderScene.mock.calls[4][0];
-    expect(renderScene.mock.calls[4][1]).toBeNull();
-    expect(elements.length).toEqual(1);
-    expect([elements[0].x, elements[0].y]).toEqual([29, 39]);
-    expect([elements[0].width, elements[0].height]).toEqual([31, 31]);
+    expect(h.appState.selectionElement).toBeNull();
+    expect(h.elements.length).toEqual(1);
+    expect([h.elements[0].x, h.elements[0].y]).toEqual([29, 47]);
+    expect([h.elements[0].width, h.elements[0].height]).toEqual([30, 50]);
```

Comment:
How is this `[30, 50]` (that is, unchanged) is beyond me.

Reply:
Since we're not currently keeping aspect ratio when holding shift, this must be wrong (in current, and previous test) :/. Would be great if we had visual confirmation of what's going on.